lemur/certificates/cli.py

```diff
@@ -139,14 +140,16 @@ def request_rotation(endpoint, certificate, message, commit):
                                                                  "endpoint": str(endpoint.dnsname)})
 
 
-def request_reissue(certificate, commit):
+def request_reissue(certificate, notify, commit):
     """
     Reissuing certificate and handles any exceptions.
     :param certificate:
+    :param notify:
     :param commit:
     :return:
     """
     status = FAILURE_METRIC_STATUS
+    notify = certificate.notify
```

Comment:
I am trying to understand what is the purpose if `notify`, if it is overwritten by `certificate.notify`.

Are we thinking more like
```
notify = certificate.notify and notify
```
basically, as long as both certificate notification is on, and the cli call is requesting notify, we send out additional notifications? 

or do we want it to be just controlled via the `notify` input parameter?

Reply:
You're right, this is a mistake. I originally didn't have it configurable (they were always enabled, assuming `certificate.notify` was enabled) but I later added the config option and passed it via the same variable name (oops!). So, my intention was `notify = certificate.notify and notify` - but I'm open to suggestions if you disagree.

Reply:
I think it might be a good case to _ignore_ `certificate.notify` since honoring it risks unnoticed failure. That said, _not_ honoring it might confuse people. Overall, I'm thinking that sending regardless of `certificate.notify` is probably the safest option to avoid failures.

Reply:
I also prefer this option, since it gives the flexibility to control this type of notification via the UI.